runtime/src/accounts.rs

```diff
@@ -4257,4 +4261,64 @@ mod tests {
         test(tx_set_limit_99, &feature_set, &result_requested_limit);
         test(tx_set_limit_0, &feature_set, &result_invalid_limit);
     }
+
+    #[test]
+    fn test_load_accounts_too_high_prioritization_fee() {
+        solana_logger::setup();
+        let lamports_per_signature = 5000_u64;
+        let request_units = 1_000_000_u32;
+        let request_unit_price = 2_000_000_000_u64;
+        let prioritization_fee_details = PrioritizationFeeDetails::new(
+            PrioritizationFeeType::ComputeUnitPrice(request_unit_price),
+            request_units as u64,
```

Comment:
unrelated to the scope of this PR - it seems odd that this takes `request_units` as a `u64`, but `set_compute_unit_limit` takes it as a `u32`

Reply:
yea, there is inconsistency between https://github.com/solana-labs/solana/blob/master/sdk/src/compute_budget.rs#L40 and https://github.com/solana-labs/solana/blob/master/program-runtime/src/compute_budget.rs#L34